src/renderer/components/dock/dock.tsx

```diff
@@ -71,6 +76,7 @@ export class Dock extends React.Component<Props> {
         {isInstallChartTab(tab) && <InstallChart tab={tab} />}
         {isUpgradeChartTab(tab) && <UpgradeChart tab={tab} />}
         {isTerminalTab(tab) && <TerminalWindow tab={tab} />}
+        {isPodLogsTab(tab) && <PodLogs tab={tab} />}
```

Comment:
This is less of a note about this PR and more about the design of the dock in general. I really feel that the dock shouldn't really care/know about what sort of tabs it has. But that is for another time.